docs/source/getting_started/main_concepts.md

```diff
@@ -0,0 +1,124 @@
+# 🧠 Main Concepts
+
+## Overview
+Training a model with Streaming happens in 2 main steps:
+1. Convert your data to a StreamingDataset-compatible format
+2. Distributed model training with StreamingDataset
+
+Let's cover the key concepts in this process.
+
+## Dataset conversion
+Raw data samples need to be processed into a **stream**, or set of **shard** files, that allow for fast random access during training. Streaming supports the following file formats:
+* MDS (most performant)
+* XSV
+* JSONL
+
+A **shard** is a file, compatible with Streaming, that contains samples that are ready for training.
+
+A **stream** is a collection of shard files.
+
+The diagram below shows how raw data samples are converted to MDS shards using {class}`streaming.MDSWriter` objects.
+
+<img src="../_static/images/mds_writing.png" alt="MDS Writing" width="800"/>
```

Comment:
Instead of an illustration of the index.json file, I instead included a snippet that shows how the index.json file is structured internally to hold dataset + shard metadata. I think that works better